.github/workflows/build-and-test-arm.yaml

```
@@ -6,7 +6,15 @@ on:
   workflow_dispatch:
 
 jobs:
+  pre_condition:
+    name: Don't run scheduled jobs on forks
+    runs-on: ubuntu-latest
+    if: ${{ (github.event_name != 'schedule') || (github.event_name == 'schedule' && github.repository == 'autowarefoundation/autoware.universe') }}
```

I think we can consider this.

```suggestion
    if: ${{ (github.event_name != 'schedule') || (github.event_name == 'schedule' && github.repository_owner == 'autowarefoundation') }}
```